python/ray/data/read_api.py

```diff
@@ -1018,6 +1018,7 @@ def read_parquet_bulk(
 def read_json(
     paths: Union[str, List[str]],
     *,
+    include_paths: bool = False,
```

Comment:
Hi, you also need to update https://github.com/ray-project/ray/blob/master/python/ray/data/datasource/json_datasource.py as well. See image data source as an example - https://github.com/ray-project/ray/blob/master/python/ray/data/datasource/image_datasource.py#L95-L98 .